### DISCUSSION TITLE
Charts

### DISCUSSION THREAD

**Discussion by Darathor17** [mantinedev/mantine#5522] [Feature requests]
Hi! 

First of: great job on mantine/charts! 

I have a few questions for you (and possibly features request): 

- Do you think we could get animations? I see in the codebase you disable them (`isAnimationActive={false}`). Maybe it could be an optional option?

- Is there a way to pass props directly to `<Bar/>`? At the moment we can pass props to `<BarChart/>` if I'm not mistaken? (for instance we could manage radius for bars!)

- For BarChart 'color' are required in `series`. Since we can use the `fill` option in the data directly, maybe make it optional with a default value? for now I have to either put a random color (that is not applied) or simply `''`

Thx a ton!

Cheers

**Comment by highviolet** [comment]
Another suggestion: Using the underlying ReCharts library it is possible to have mixed line types in a line chart.
It would be useful enable the same within mantine, e.g. by adding another property to LineChartSeries.

**Comment by ReedME** [comment]
Would love to be able to allow the recharts animation optionally!

**Comment by atumas-bananamilk** [comment]
Any update on animations?
@rtivital

**Comment by LenniM** [comment]
Hey! You can enable animations through accessing the RechartsProps (for example pieProps) and setting isAnimationActive to true. Example:
```
<PieChart
  data={[
	  { name: "Test1", value: 2, color: "teal.6" },
	  { name: "Test2", value: 1, color: "indigo.6" },
  ]}
  withLabelsLine
  labelsPosition="outside"
  labelsType="value"
  withLabels
  pieProps={{ isAnimationActive: true, dataKey: "value" }}
/>
```

**Reply by atumas-bananamilk** [reply]
Works perfect!
Link: https://recharts.org/en-US/api/Bar#isAnimationActive
